api-reference/10 UI Components/GridBase/3 Methods/abortAIColumnRequest(columnName).md

```diff
@@ -3,10 +3,10 @@ id: GridBase.abortAIColumnRequest(columnName)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Aborts an AI column's requests.
```

Comment:
It's always a tough situation when we use a noun phrase or a possessive phrase and different parts require different articles. In this case it's "an AI column", but "requests" needs no article. But usually, in such phrases, the article applies to the very last noun. So it seems like you're saying "an requests". So either we go without article: 

```suggestion
Aborts AI column requests.
```

Or we can split this into two parts, each with their own article: 

 ```suggestion
Aborts requests initiated by an AI column.
```